src/components/control/Chip/Chip.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import { StoryFn } from '@storybook/react';
+import React, { useState } from 'react';
+
+import './Chip';
+
+export default {
+  component: 'evg-chip',
+  parameters: {
+    docs: {
+      description: {
+        component: `Chips are used to filter content by category or type. They can be used in groups to represent a selection of options or tag links.`,
+      },
+    },
+  },
+  argTypes: {
+    variant: {
+      control: {
+        type: 'radio',
+      },
+      options: ['light', 'dark'],
+    },
+  },
+};
+
+export const Chip: StoryFn = (args) => {
+  return (
+    <evg-chip {...args}>
+      <button type="button">Chip</button>
+    </evg-chip>
+  );
+};
+
+Chip.args = {
+  variant: 'light',
+  selected: false,
+};
+
+export const ControlledChip: StoryFn = (args) => {
+  const [selected, setSelected] = useState(false);
+  return (
+    <evg-chip {...args} selected={selected}>
+      <button type="button" onClick={() => setSelected(!selected)}>
+        Chip Controlled
+      </button>
+    </evg-chip>
+  );
+};
+
+ControlledChip.parameters = {
+  docs: {
+    description: {
+      story: `Chips state can be controlled by the <code>selected</code> prop`,
+    },
+  },
+};
+
+export const CheckboxChip: StoryFn = () => (
```

Comment:
Oh there you go you have a checkbox one anyway so maybe we should just change selected to be button aria-pressed